src/NuGet.Core/NuGet.LibraryModel/LibraryDependency.cs

```diff
@@ -22,6 +22,11 @@ public string Name
             get { return LibraryRange.Name; }
         }
 
+        /// <summary>
+        /// True if the PackageReference is added by the SDK and not the user.
+        /// </summary>
+        public bool AutoReferenced { get; set; }
```

Comment:
I'm going to keep everything consistent with the project property name, I think this name works as is.